book/website/community-handbook/consistency/consistency-formatting.md

```diff
@@ -0,0 +1,194 @@
+If no ALT text is provided with an image, these users will be unable to understand the purpose of the image.
+
+When writing ALT text, remember to:
+- **Be descriptive** - Adequately describe the image using its content and context for guidance. 
+In doing so, there is no need to "announce" an image in your description (for example, using "illustration of" or "picture of") since screen readers will already do this.
+- **Keep it as short as possible** - Although a long description may be necessary for some images, it is better to keep them as short as possible. 
+This ensures that the descriptions are easy to understand.
+
+Please note that images included in _The Turing Way_ book should be less than 1MB.
+This allows the book load faster, especially for readers who may have slow internet connections.
+
+Please refer to the {ref}`style guide <ch-style-figures>` for examples on formatting images using `MyST` and adding ALT text to them.
+When including images in your contributions, it may be better to avoid the height parameter as the wrong value could make your image appear distorted on mobile devices.
+You should always check how your image looks in the Netlify preview of the book when you make a PR.
+
+(ch-consistency-formatting-sr)=
+## Soft Requirements
+
+Soft requirements help improve the overall look and feel of _The Turing Way_.
+When effected, these checks may go unnoticed, but they also make _The Turing Way_ a polished piece of work.
+Soft requirements that deal with _The Turing Way's_ formatting include: 
+
+(ch-consistency-formatting-sr-one)=
+### Check 1: Ensure that the names of chapters/subchapters are short and map exactly to how they are titled in the `_toc.yml`
+
+Some chapters and subchapters in _The Turing Way_ do not match their corresponding references in the book-wide table of contents that appears on the left of the webpage.
+This may be confusing for users, especially when the chapter/subchapter's reference in the table of contents significantly varies from the chapter/subchapter's name.
+
+```{figure} ../../figures/mismatched_title_toc.png
+---
+name: mismatched_title_toc
+alt: A subchapter whose title differs from its reference in the table of contents. The title of the subchapter is 'Using Spreadsheets for Research Data', however in the table of contents, it is referred as 'Data Organisation in Spreadsheets'.
+---
+The title of this subchapter is 'Using Spreadsheets for Research Data', however the table of content refers to the same file as 'Data Organisation in Spreadsheets'.
+```
+
+In ensuring that _The Turing Way's_ content passes this check, one recommendation to follow is to keep the titles short.
+If a chapter's title and its reference in the table of contents differ, make the shorter of the two the chapter's title, and update the `_toc.yml` if necessary. 
```

Comment:
Is this more for exisiting chapters? So if readers come across this mismatch?

Reply:
I added a new sentence to capture the use case of people writing new chapters